README.en.md

```diff
@@ -932,6 +933,31 @@ outbound_proxy_host = site.com
 outbound_proxy_port = 8080
 ```
 
+#### `hreflang_x_default_lang`
+Configures the language code used to generate link tag with `hreflang="x-default"` attribute.
+
+The language code used must be in `supported_langs`. If the language code is invalid, link tag will not be inserted.
+
+If a link tag with `hreflang="x-default"` attribute exists, this setting does nothing.
```

Comment:
When customer uses `custom_domain` pattern, there is a case we need to overwrite link tag with `hreflang="x-default"` when customer has config to change their default lang URL.

Reply:
For now, I agree with current spec.